src/Cake.Frosting.Template/templates/cakefrosting/build.ps1

```diff
@@ -0,0 +1,2 @@
+dotnet run --project build/Build.csproj -- $args
```

Comment:
We no longer want to download .NET as part of the bootstrapper? What's the reason for this?
Otherwise https://github.com/cake-build/cake/pull/2960 would be a solution using global.json.

Reply:
Should the standard bootstrappers install .NET?
Maybe a good thing to keep these as simple as possible.
The template itself already requires .NET Core / .NET 5 to be installed.
I kinda like the simplicity of what's suggested here.
Maybe better to document `dotnet new globaljson` and have more advanced bootstrappers in our resources repo?

Reply:
I was never team "Install .NET in bootstrapper" 😉 It's just what most people seem to prefer nowadays, and I wanted to bring it up to discussion, since it's quite a hidden change. Template requiring .NET to be installed probably isn't an argument since people / machine running build in most cases is not the same which did run the template.

@gep13 Did you already though about how bootstrappers in resources repo should look like for .NET tool and .NET Core regarding installation of .NET?

Reply:
@pascalberger I haven't come to a final decision about how these should look yet.  I am still thinking that installing .Net locally into the repository folder, based on what is in the global.json is a good idea though.  I liked what was done with the `jsonfile` parameter, and I thought about putting an if statement round that to either pass it in or not, based on the global.json file being present.

Reply:
I don't think that the frosting bootstrapper should install .NET. It should just be a easy way to run a build which exists in another directory.

Reply:
Then it maybe should push / change / pop directory so it's executed in script directory?

Reply:
That's not needed since the working directory already is the script directory. Working directory should imo not be set by a script but be explicitly stated by the person calling the script